pkgs/development/libraries/libpam-wrapper/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, fetchgit
+, cmake
+, openpam
+, enablePython ? false
+, python ? null
+}:
+
+with stdenv.lib;
```

Comment:
`with` statement makes it harder to reason about code. Prefer using qualified names, `let` with `inherit` or narrow the scope of `with` as much as possible.